app/build.gradle.kts

```diff
@@ -86,21 +86,12 @@ android {
         getByName("debug") {
             isMinifyEnabled = false
 
-            versionNameSuffix = "-dev"
-            applicationIdSuffix = ".dev"
+            applicationIdSuffix = ".debug"
 
             signingConfig = signingConfigs.getByName("debug")
         }
     }
 
-    compileOptions {
-        sourceCompatibility = JavaVersion.VERSION_17
-        targetCompatibility = JavaVersion.VERSION_17
-    }
-
-    kotlinOptions {
-        jvmTarget = JavaVersion.VERSION_17.toString()
-    }
```

Comment:
As of AGP 8.1.0, when using the gradle toolchain, this configuration is not necessary